GitUI/BuildServerIntegration/BuildServerWatcher.cs

```diff
@@ -155,7 +155,7 @@ public void CancelBuildStatusFetchOperation()
                             byte[] unprotectedData = ProtectedData.Unprotect(protectedData, null,
                                 DataProtectionScope.CurrentUser);
                             using MemoryStream memoryStream = new(unprotectedData);
-                            ConfigFile credentialsConfig = new("", false);
+                            ConfigFile credentialsConfig = new("");
```

Comment:
[nit] Not terribly important, but should this be?
```suggestion
                            ConfigFile credentialsConfig = new(fileName: "");
```

Here and in few other places

Reply:
Added for the three occurrences (excluding tests). It is Config**File**, so indeed "not terribly important".